api/mymonero_rest.md

```diff
@@ -0,0 +1,391 @@
+| view_key         | binary         | View key bytes                   |
+| amount           | uint64-string  | XMR send amount                  |
+| mixin            | uint32         | Minimum mixin for source output  |
+| use_dust         | boolean        | Return all available outputs     |
+| dust_threshold * | uint64-string  | Ignore outputs below this amount |
+
+> If the total received outputs for the address is less than `amount`, the
+> server shall return a HTTP 400 "Bad Request" error code.
+
+**Response** object
+
+|    Field   |        Type       |                Description              |
+|------------|-------------------|-----------------------------------------|
+| per_kb_fee | uint64            | Estimated network fee                   |
+| amount     | uint64-string     | The total value in outputs              |
+| outputs    | array of output's | Outputs possibly available for spending |
+
+#### `import_request`
+Request an account scan from the genesis block.
+
+**Request** object
+
+|   Field  |      Type      |      Description        |
+|----------|----------------|-------------------------|
+| address  | base58-address | Address to create/probe |
+| view_key | binary         | View key bytes          |
+
+**Response** object
+
+|       Field        |      Type      |           Description            |
+|--------------------|----------------|----------------------------------|
+| payment_address *  | base58-address | Payment location                 |
+| payment_id *       | binary         | Bytes for payment_id tx field    |
+| import_fee *       | uint64-string  | Fee required to complete request |
+| new_request        | boolean        | New or existing request          |
+| request_fulfilled  | boolean        | Indicates success                |
+| status             | string         | Custom message                   |
+
+> `payment_id`, `import_fee`, and `payment_address` may be omitted if the
+> client does not need to send XMR to complete the request.
+
+#### `login`
+Check for the existence of an account or create a new one.
+
+**Request** object
+
+|       Field       |      Type      |           Description            |
+|-------------------|----------------|----------------------------------|
+| address           | base58-address | Address to create/probe          |
+| view_key          | binary         | View key bytes                   |
+| create_account    | boolean        | Try to create new account        |
+| generated_locally | boolean        | Indicate that the address is new |
+
+> The view key bytes are required even if an account is not being created, to
+> prevent metadata leakage.
+
+> If the server does not allow account creations, HTTP 501 "Not Implemented"
+> error must be returned.
+
+> If approval process is manual, a successful HTTP 200 OK and response object
+> must be returned. Subsequent requests shall be HTTP 405 "Forbidden" until
```

Comment:
I think it could be more useful HTTP 403. 

Details about HTTP 403 [RFC 2616](https://tools.ietf.org/html/rfc2616) :
>  The server understood the request, but is refusing to fulfill it. Authorization will not help and the request SHOULD NOT be repeated. If the request method was not HEAD and the server wishes to make public why the request has not been fulfilled, it SHOULD describe the reason for the refusal in the entity. If the server does not wish to make this information available to the client, the status code 404 (Not Found) can be used instead.